.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,49 @@
+name: CI Workflow
+
+on:
+  [push, workflow_dispatch]
```

Comment:
This looks different than what I'd seen and copied from mgmt-api: 

```

on: 

  push:

    branches: [ '**' ]

  workflow_dispatch:

```

It's possible they behave the same, but I'm not totally sure how this syntax works. Do you know/did you do it this way for another reason?

Reply:
I replaced that syntax from what I originally had (looked more like `hri-mgmt-api`) based off of the [ci-workflow.yaml](https://github.com/Alvearie/hri-api-spec/blob/develop/.github/workflows/ci-workflow.yaml) in `hri-api-spec`.  I don't mind using the `hri-mgmt-api` version, I'll go ahead and replace it.